microsoft-edge/web-platform/password-reveal.md

```diff
@@ -0,0 +1,63 @@
+
+![A password field with dots hiding entered characters. An icon shaped like an eye is displayed within the right-hand side of the field.](../media/web-platform/password-reveal/password-eye.png)
+
+_The password reveal button appears upon text entry._
+
+![The eye icon in the same password field now has a stroke through it, and the characters are revealed.](../media/web-platform/password-reveal/password-toggled.png)
+
+_Invoking the button turns the obfuscated dots into whatever the user typed into the field._
+
+By default, the password reveal button is inserted into the Shadow DOM of all HTML inputs of `type="password"`. As of Microsoft Edge v87, users or [enterprises](https://docs.microsoft.com/deployedge/microsoft-edge-policies#passwordrevealenabled) may disable this feature globally. Web designers and developers should expect most Microsoft Edge users to have the default experience.
+
+## Removing the browser’s password reveal control
+
+Web developers can completely remove the browser’s password reveal control (e.g. in favor of their own) by targetting the `::-ms-reveal` pseudo element:
+
+```
+::-ms-reveal {
+  display: none;
+}
+```
+
+However, we recommend that most websites take advantage of the browser’s password reveal button, as the control’s behavior has important [security measures](#visibility-of-the-control) built in.
+
+## Customizing control styling
+
+Web developers can instead tweak the button’s styling so it better matches the website’s visual language. For example:
+
+```
+::-ms-reveal {
+  border: 1px solid transparent;
+  border-radius: 50%;
+  box-shadow: 0 0 3px currentColor;
+}
+```
+
+Some notes on styling:
+
+* The browser’s “eye” iconography is implemented as a background image. To add a background color to the button, use the CSS `background-color` property instead of the `background` shorthand property.
+* Web developers may adjust the size and scale of the password reveal button, but the button will be clipped by the bounds of the password input control (if applicable).
+* There are currently no state selectors available to style the button’s “toggled” state.
+
+## Visibility of the control
+
+While trying out the password reveal button, it may be helpful to understand in which circumstances the button is displayed. The password reveal control is not available until the user enters text into the field, and as added security precautions:
+
+* The button is removed if focus moves from the field.
+* The button is removed if scripts modify the password field.
+* If the button is removed at any point, the entire contents of the password field must also be removed in order to bring the button back (preventing someone from making a minor adjustment in order to view the password).

```

Comment:
Regarding removal of the button - do we mean that the dev removes the class at runtime? If yes, can we add that clarification please?

Reply:
No, I'm referring to the removals described in the previous bullets. Any suggestions on clarifying?

Reply:
I marked it up to be a bit cleaner and more readable.

Reply:
Did you push your changes? I don't think I see them.